### PR TITLE
Added intercom bubble to verification

### PR DESCRIPTION
**Pull request by fpsbernardo**

### CODE REVIEW

File: src/verification/address/AddressComponent/AddressForm/AddressForm.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
 import { VERIFICATION_CHOOSE_ROUTE } from '../../../constants';
 import { addressFormSubmitHandler } from '../../addressRoutine';
 import { type Address, type AddressForm } from '../../addressState';
+import IntercomBubble from '../../../../../src/landing/IntercomBubble';
```
Comment (v3rm0n) [inline src/verification/address/AddressComponent/AddressForm/AddressForm.js]: Going up one too many level, src should not be here

File: src/verification/confirm/Confirm.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { FormFeedback } from '../../ui/form';
 import confirmRoutine from './confirmRoutine';
 import EUIcon from '../icon/EUIcon';
 import { VERIFICATION_SELFIE_VERIFICATION_ROUTE } from '../constants';
+import IntercomBubble from '../../../src/landing/IntercomBubble';
```
Comment (v3rm0n) [inline src/verification/confirm/Confirm.js]: Same here

File: src/verification/intro/Intro.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ import idLogo from './img/id_logo.png';
 import selfieLogo from './img/selfie_logo.png';
 import EUIcon from '../icon/EUIcon';
 import { userIsVerified } from '../../redux/selectors';
+import IntercomBubble from '../../../src/landing/IntercomBubble';
```
Comment (v3rm0n) [inline src/verification/intro/Intro.js]: And here

File: src/verification/profile/Profile.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ import {
   VERIFICATION_ADDRESS_ROUTE,
   VERIFICATION_INTRO_ROUTE,
 } from '../constants';
+import IntercomBubble from '../../../src/landing/IntercomBubble';
```
Comment (v3rm0n) [inline src/verification/profile/Profile.js]: And here

File: src/verification/verificationFiles/address/AddressVerification.js [hunk 5]
```diff
@@ -19,6 +19,7 @@ import {
 } from '../../constants';
 import verificationFileUploader from '../verificationFileUploader';
 import EUIcon from '../../icon/EUIcon';
+import IntercomBubble from '../../../../src/landing/IntercomBubble';
```
Comment (v3rm0n) [inline src/verification/verificationFiles/address/AddressVerification.js]: etc

**Comment by fpsbernardo** [comment]
@JordanValdma

**Pull request merged**